### PR TITLE
Adding and Removing Layouts Example Additions.

### PR DESCRIPTION
**Pull request by joeransegnola**
Adding a couple of buttons here to show how it'll be possible to remove an existing layout, persist it locally in LocalStorage, and then inject a new snapshot from a file as well.

### CODE REVIEW

File: how-to/web-layout/public/platform/provider.html [hunk 1]
```diff
@@ -19,14 +19,17 @@ <h1>Web Multiple Layout Example</h1>
 				<h1 class="tag">Demonstrate web interop usage with multiple layouts</h1>
 			</div>
 			<div class="row middle gap20">
-				<button id="swap-layouts">Swap Layouts</button>
+				<button id="add-layout">+</button>
```
Comment (johnman) [inline how-to/web-layout/public/platform/provider.html:22]: If you want joe you can take the + SVG from the support context and intents example as it is a bit bigger.

**Review by johnman — CHANGES_REQUESTED** [review]
Hi Joe,

Can you update your branch from /vnext and then issue the pr against the vnext branch. This branch will then be promoted to main. The vnext branch is also using 0.38.55 so you should be able to re-run it.

File: how-to/web-layout/public/platform/provider.html (hunk 1 above)
Comment (joeransegnola) [reply how-to/web-layout/public/platform/provider.html]: Couldn't find the svg, but added the word "Add" to make it more obvious

**Review by johnman — APPROVED** [review]

**Review by johnman — COMMENTED** [review]
Hi Joe,

I just did a revert so you will need to re-issue the PR.

If you add a layout, delete it and then add a layout again it won't let you do it as it believes a layout with that id still exists so that just needs to be resolved.

**Pull request merged**